README.md

```diff
@@ -1,10 +1,155 @@
+  up is building a container specific VMM. In doing so, the container VMM can
+  reuse most of the rust-vmm components and just build the glue around these
+  components as well as an appropriate API for interacting with the container.
+- **Security & Testability**. Having independent components corresponding to
+  virtualization components makes it easier to run fuzzing on individual
+  packages. The crate we want to focus on in the near future is
+  [vm-virtio](https://github.com/rust-vmm/vm-virtio) which provides a
+  [virtio](https://www.oasis-open.org/committees/tc_home.php?wg_abbrev=virtio)
+  device implementation. We want to keep a high standard in terms of testing
+  as these virtualization packages are going to be used in production by
+  multiple projects. Each component is individually tested with a set of
+  common integration tests that are responsible for running the unit tests,
+  computing the coverage, coding style and other lint checks. In the future
+  we will test the integration of the rust-vmm components by providing a
+  reference VMM implementation.
+- **Clean interface**. With shared crates you also have to think about an
+  interface flexible enough to be used by multiple VMMs. We have multiple
+  rounds of design reviews up to the point we are confident the interface is
+  clean and re-usable by other projects.
+
+## rust-vmm components
+
+Keep in mind that this status might not always be up to date. For the latest
+status please check individual repositories.
+
+**Components with PRs in fly**
+
+These are the empty repositories that have PRs waiting to be merged.
+TODO
+
+**Components in Development:**
+
+These are packages for which we have most of the code out there, but
+there might be some missing requirements for publishing them on crates.io.
+For the full list of these requirements please check the
+[Publishing on crates.io - Requirements list](#publishing-on-crates.io---requirements-list)
+section.
+
+- [vm-memory](https://github.com/rust-vmm/vm-memory): abstractions over a
+  Virtual Machine memory.
+
+**Components Published on [crates.io](crates.io):**
+
+These are production-ready components that can be pulled as dependencies
+directly from crates.io.
+
+- [kvm-bindings](https://crates.io/crates/kvm-bindings): Rust FFI bindings
+  to KVM generated using [bindgen](https://crates.io/crates/bindgen).
+- [kvm-ioctls](https://crates.io/crates/kvm-ioctls): Safe wrappers over the
+  KVM API.
+
+## How to contribute
+
+### Adding a new virtualization component
+
+If you have an idea about a component that could be a great fit for rust-vmm,
+please open a
+[review request issue](https://github.com/rust-vmm/community/issues/new?assignees=&labels=&template=repository-review-request.md&title=Repository%2FCrate+Review+Request).
+The issue is a pitch for the component you want to add so in it you should
+write details about the purpose of the repository, how will it be used and
+other information that you find relevant.
```

Comment:
"The issue must provide the explanations and justifications behind the need for a new component, highlighting the purpose of this new repository, how....."